CriticalMass/Sequence+Additions.swift

```diff
@@ -4,9 +4,9 @@
 import Foundation
 
 extension Sequence {
-    func sorted<T: Comparable>(by keyPath: KeyPath<Element, T>) -> [Element] {
-        sorted { a, b in
-            a[keyPath: keyPath] < b[keyPath: keyPath]
+    func sorted<T: Comparable>(by keyPath: KeyPath<Element, T>, sortFunction: ((T, T) -> Bool) = (<)) -> [Element] {
```

Comment:
Maybe this should be called sortOperator and I think it could have a default implementation of < since that feels like the most natural ordering

Reply:
I agree with the default implementation, however we have currently more calls were we are using > and I don't want to further grow the scope of this PR. So I'll suggest keeping it as it is for this PR.